guides/source/active_record_postgresql.md

```diff
@@ -504,7 +503,7 @@ irb> device.id
 => "814865cd-5a1d-4771-9306-4268f188fe9e"
 ```
 
-NOTE: `gen_random_uuid()` (from `pgcrypto`) is assumed if no `:default` option was
```

Comment:
This line should not be changed because Rails supports PostgreSQL 9.3 or higher. PostgreSQL versions between 9.3 and 12 need `pgcrpyto`.